backend/backend_config.py

```diff
@@ -8,7 +8,7 @@
 top_k = 10
 
 # Protein file path relative to root.
-protein_path = "data/pdb_data_seq.csv"
+protein_path = "data/pdb_data_seq_culled.csv"
```

Comment:
I wouldn't change the name, and just overwrite `pdb_data_seq.csv` with the culled data.

Reply:
Good shout will do! This is WIP so ignore for now. I'm gonna do something a little more elegant!